Marlin/src/feature/bedlevel/abl/bbl.cpp

```diff
@@ -118,18 +118,18 @@ void LevelingBilinear::set_grid(const xy_pos_t& _grid_spacing, const xy_pos_t& _
  */
 void LevelingBilinear::extrapolate_unprobed_bed_level() {
   #ifdef HALF_IN_X
-    constexpr uint8_t ctrx2 = 0, xend = GRID_MAX_POINTS_X - 1;
+    TERN(PROUI_GRID_PNTS, const, constexpr) uint8_t ctrx2 = 0, xend = GRID_MAX_POINTS_X - 1;
   #else
-    constexpr uint8_t ctrx1 = (GRID_MAX_CELLS_X) / 2, // left-of-center
-                      ctrx2 = (GRID_MAX_POINTS_X) / 2,  // right-of-center
+    TERN(PROUI_GRID_PNTS, const, constexpr) uint8_t ctrx1 = (GRID_MAX_CELLS_X) / 2,  // left-of-center
+                      ctrx2 = (GRID_MAX_POINTS_X) / 2, // right-of-center
                       xend = ctrx1;
   #endif
 
   #ifdef HALF_IN_Y
-    constexpr uint8_t ctry2 = 0, yend = GRID_MAX_POINTS_Y - 1;
+    TERN(PROUI_GRID_PNTS, const, constexpr) uint8_t ctry2 = 0, yend = GRID_MAX_POINTS_Y - 1;
   #else
-    constexpr uint8_t ctry1 = (GRID_MAX_CELLS_Y) / 2, // top-of-center
-                      ctry2 = (GRID_MAX_POINTS_Y) / 2,  // bottom-of-center
+    TERN(PROUI_GRID_PNTS, const, constexpr) uint8_t ctry1 = (GRID_MAX_CELLS_Y) / 2,  // top-of-center
```

Comment:
You can't use `TERN()` with the integer define `PROUI_GRID_PNTS`.

Reply:
wait im confused, `PROUI_GRID_PNTS` is enabled by being defined as 1 - like "true". just as many other examples use `TERN` or `TERN_(HAS_....)`, as in `HAS_whatever` also defined as 1